test/extensions/filters/http/common/fuzz/uber_filter.cc

```diff
@@ -73,7 +73,11 @@ void UberFilterFuzzer::fuzz(
         proto_config, factory_context_.messageValidationVisitor(), factory);
     // Clean-up config with filter-specific logic before it runs through validations.
     cleanFuzzedConfig(proto_config.name(), message.get());
-    cb_ = factory.createFilterFactoryFromProto(*message, "stats", factory_context_).value();
+    auto cb_or = factory.createFilterFactoryFromProto(*message, "stats", factory_context_);
+    if (!cb_or.ok()) {
```

Comment:
I think this change will now be a silent failure.
Please use `THROW_IF_STATUS_NOT_OK` so it will be caught by the catch below and a log-message will appear.
(later if/when the catch is removed from the code, the remover will need to ensure that the error is handled gracefully.